### DISCUSSION TITLE
Sound when navigating

### DISCUSSION THREAD

**Discussion by spaceman66** [EmiyaSyahriel/CrossLauncher#74] [Ideas]
Have an option to have system sounds when navigation through menus, like the psp had.

**Comment by EmiyaSyahriel** [comment]
It's already there, [Here is the tutorial](https://github.com/EmiyaSyahriel/CrossLauncher/wiki/Customization#ui-sound-effect), though you're going to need a File Manager that is capable to explore and is allowed to modify the files inside of `/storage/emulated/0/Android/data/id.psw.vshlauncher/files` or `/storage/emulated/0/Android/media/id.psw.vshlauncher/files` directory.

Or, should we make it possible to do from UI instead?

**Reply by spaceman66** [reply]
I think it would be cool to have as a hi feature. But it's up to you.

**Reply by EmiyaSyahriel** [reply]
Issue entry made : #77
